modules/reference/pages/command/securityUtility-encode.adoc

```diff
@@ -27,6 +27,14 @@ If your password includes special characters, you must escape each special chara
 
 For more information about limits to password encryption, see xref:ROOT:password-encryption.adoc[Password encryption limitations].
 
+To enable the `securityUtility` tool to create LTPA keys with the `createLTPAKeys` command on FIPS 140-3 enabled servers, export the following environment variable before you run the tool:
```

Comment:
should this be 

To enable the `securityUtility` tool to encode passwords with the `encode` command...